nx/exercises/exercises-1-20.livemd

```diff
@@ -0,0 +1,196 @@
+# Exercises: 1-20
+
+```elixir
+Mix.install([{:nx, github: "elixir-nx/nx", sparse: "nx"}])
+```
+
+## Introduction
+
+Inspired by the Python notebook _100 Numpy Exercises_:
+
+https://www.kaggle.com/code/utsav15/100-numpy-exercises/notebook
+
+## 1-10
+
+1. Install Nx in a Livebook (★☆☆)
+
+```elixir
+Mix.install([{:nx, github: "elixir-nx/nx", sparse: "nx"}])
+```
+
+2. Print the Nx version and the configuration (★☆☆)
+
+```elixir
+version = Application.spec(:nx, :vsn)
+# I'm not sure if there's an equivalent for "configuration"
+```
+
+3. Create a 1-D tensor of 10 zeros (★☆☆)
+
+```elixir
+# I'm not sure what the best way to do this is.
+# I think Nx could use a function like: `Nx.fill(value, shape)`
+
+# 1.
+#    0 |> List.duplicate(10) |> Nx.tensor()
+
+# 2.
+#    [0] |> Nx.tensor() |> Nx.tile([10])
+
+# 3.
+#    i = Nx.iota({10})
+#    Nx.subtract(i, i)
+
+0 |> List.duplicate(10) |> Nx.tensor()
+```
+
+4. How to find the memory size of any tensor (★☆☆)
+
+```elixir
+Nx.byte_size(Nx.tensor([[1, 2, 3], [4, 5, 6]]))
```

Comment:
Should also mention `flat_size` for vectorized tensors

Reply:
We may want consecutive exercises for `size`, `flat_size`, and `byte_size`. It'd highlight how vectorization changes the answers.

Reply:
Yeah, I think it's a good idea!